src/Components/Crafting/CraftingButton.js

```diff
@@ -1,7 +1,29 @@
 import React from 'react';
+import CraftingWindow from './CraftingWindow';
 import './CraftingButton.css';
+import GeneralButton from 'Components/GeneralComponents/GeneralButton';
 import { craftingButtonTexts } from 'Texts/gameplayTexts';
+import { mountComponents, unmountComponents } from 'Redux/Slices/MountedComponents'; //???
+import { useDispatch, useSelector } from 'react-redux';
 
 export default function CraftingButton() {
-  return <div id="craftingButton">{craftingButtonTexts.title}</div>;
+	const dispatch = useDispatch();
+	const isCraftingWindowOpen = useSelector(state => state.MountedComponents.CraftingWindow);
```

Comment:
please change to `isCraftingWindowMounted`

Reply:
Done.